### ISSUE TITLE
swipeEnabled true breaks nested flex columns

### ISSUE THREAD

**Issue by lefloh** [react-navigation/react-navigation#1964]
### Current Behavior

Defining Columns via `flex` is broken if nested inside a `TabNavigator` with `swipeEnabled: true` in TabNavigatorConfig.

```javascript
// @flow
import React from 'react';
import { AppRegistry, StyleSheet, Text, View } from 'react-native';
import { TabNavigator } from 'react-navigation';

const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    marginTop: 20,
  },
  row: {
    flex: 1,
    flexDirection: 'row',
    borderBottomColor: 'gray',
    borderBottomWidth: 1,
  },
  left: {
    flex: 3,
    backgroundColor: 'green',
  },
  right: {
    flex: 1,
    backgroundColor: 'red',
  },
});

function DemoScreen() {
  return (
    <View style={styles.wrapper}>
      <View style={styles.row}>
        <View style={styles.left}>
          <Text>foo</Text>
        </View>
        <View style={styles.right}>
          <Text>bar</Text>
        </View>
      </View>
      <View style={styles.row}>
        <View style={styles.left}>
          <Text>foo</Text>
        </View>
        <View style={styles.right}>
          <Text>bazbaz</Text>
        </View>
      </View>
    </View>
  );
}

const DemoApp = TabNavigator(
  {
    Demo: {
      screen: DemoScreen,
    },
  },
  {
    swipeEnabled: true,
  }
);

AppRegistry.registerComponent('DemoApp', () => DemoApp);
```

![tabs](https://user-images.githubusercontent.com/1819274/27444933-4c6caf3e-5778-11e7-8406-1e0dac2f6610.png)

**The columns have the same width if `swipeEnabled=false`**

### Expected Behavior

A simple workaround is setting a fixed width but columns should not be affected by `swipeEnabled`. 

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation |  1.0.0-beta.11
| react-native     |  0.44.2
| node             | 8.1.2
| npm or yarn      | yarn 0.24.6

**Comment by kelset** [comment]
Pinging OP @lefloh since this issue has not been active for a while, and it's related to an old version of the lib.

Please let me know if you want this to remain open; if I get no answer in the next 7 days I will close it.

**Comment by lefloh** [comment]
This is a minor issue and there's a workaround so for me it's not important to leave this open.
I just wanted to inform you that there's a strange behaviour which may probably cause other problems.
Just tested again with `1.0.0-beta.21` and the issue still exists.

**Comment by kelset** [comment]
Gotcha, I'll keep this open then.

**Comment by satya164** [comment]
This is likely an issue with React Native itself. The difference between `swipeEnabled: false` and `swipeEnabled: true` is that the latter uses a `ScrollView` on iOS.

**Comment by cnhuye** [comment]
how about "animationEnabled:true" ?

this can make the same problem. like the linking issue #3215

**Comment by brentvatne** [comment]
this appears to be resolved in react-navigation@2.0.0 and latest version of react-native: https://snack.expo.io/Skfy7PCpf
